packages/shared/src/db/schema.ts

```diff
@@ -166,6 +168,7 @@ export const groups = sqliteTable('groups', {
   id: text('id').primaryKey(),
   ...metaFields,
   isSecret: boolean('is_secret'),
+  publicOrPrivate: text('public_or_private').$type<'public' | 'private'>(),
```

Comment:
Maybe this could be `visibility`? Also, can this be combined this with `isSecret`? Or are those separate properties?

Reply:
They're separate properties, on the backend a group could actually be set with a cordon of `shut` and still not be `secret`. This gives us the three types: public, private, secret. It's annoying that it's setup this way.

Reply:
Cool, makes sense.